src/odemis/gui/cont/tabs.py

```diff
@@ -498,6 +500,13 @@ def __init__(self, name, button, panel, main_frame, main_data):
                 self
             )
 
+    def _onView(self, view):
```

Comment:
Most of the functions in this class are snake_case, so I suggest to also use that here.
```suggestion
    def _on_view(self, view):
```